code/GridFieldDetailFormPreviewExtension.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace SilverStripe\Admin;
```

Comment:
Based on everything else in the 
https://github.com/silverstripe/silverstripe-admin/tree/1/code
and 
https://github.com/silverstripe/silverstripe-admin/tree/1/code/Forms

I'm thinking it makes sense to put it in the /Forms directory along with `GridFieldPrintButtonExtension`

Reply:
Makes sense to me. I'll update the namespace to `SilverStripe\Admin\Forms` at the same time.